zombie-monitor/src/test/scala/com/broadinstitute/dsp/zombieMonitor/DeletedDiskCheckerSpec.scala

```diff
@@ -170,4 +149,26 @@ class DeletedDiskCheckerSpec extends AnyFlatSpec with CronJobsTestSuite {
       verify(mockDbReader, never()).updateDiskStatus(disk.id)
     }
   }
+
+  it should "updateDiskStatus when a call to GCP's getDisk endpoint returns no disk (isDryRun = false)" in {
```

Comment:
This test used to be tied to gcpFailureConditions, but since the other exceptions no longer trigger updateDiskStatus, this test seemed to make more sense on its own.